kbase-extension/static/kbase/js/common/ui.js

```diff
@@ -5,21 +5,12 @@ define([
     'kb_common/html',
     'base/js/namespace',
     './runtime',
-    './events',
     'google-code-prettify/prettify',
     'css!google-code-prettify/prettify.css',
-    'bootstrap'
-], function (
-    $,
-    Promise,
-    html,
-    Jupyter,
-    Runtime,
-    Events,
-    PR
-) {
+    'bootstrap',
+], function ($, Promise, html, Jupyter, Runtime, PR) {
```

Comment:
Most of the changes in this module come from Prettier.
I also moved the `loading` function to be outside of the UI factory class. It can be accessed that way as well, though.